integration-tests/src/test/java/eu/xenit/alfred/telemetry/integrationtesting/RestAssuredTestSolr.java

```diff
@@ -33,7 +33,7 @@ public static void initializeRestAssured() {
 
         final String baseURI = "https://" + System.getProperty("solr.host", "localhost");
         RestAssured.baseURI = baseURI;
-        int port = Integer.parseInt(System.getProperty("solr.tcp.8443", "8443"));
+        int port = Integer.parseInt(System.getProperty("solr.tcp.8443", "8444"));
```

Comment:
please revert local change

Reply:
As discussed, this was to run tests locally on Windows machines. Doesn't impact Linux machines, nor github actions.